Userland/Libraries/LibJS/Runtime/StringPrototype.cpp

```diff
@@ -1202,4 +1203,22 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::sup)
     return create_html(global_object, vm.this_value(global_object), "sup", String::empty(), Value());
 }
 
+// 22.1.3.10 String.prototype.localeCompare ( that [ , reserved1 [ , reserved2 ] ] ), https://tc39.es/ecma262/#sec-string.prototype.localecompare
+JS_DEFINE_NATIVE_FUNCTION(StringPrototype::locale_compare)
+{
+    auto string = ak_string_from(vm, global_object);
+    if (!string.has_value())
+        return {};
+
+    auto that_string = vm.argument(0).to_string(global_object);
+
+    // FIXME: Actually compare the string not just according to their bits.
+    if (string == that_string)
+        return Value(0);
+    if (string.value() < that_string)
+        return Value(-1);
+
+    return Value(1);
+}
```

Comment:
This is a super simple implementation but it actually passes all test262 tests.
So is this fine as a naive implementation?

Reply:
yeah this is fine, but please add this comment below `// 22.1.3.10 ...`

```
// NOTE: This is the minimum localeCompare implementation for engines without ECMA-402.
```

so that we'll have an easier time finding all the functions that need updating when we implement ECMA-402.